base/server/etc/default.cfg

```diff
@@ -243,7 +243,7 @@ pki_enable_on_system_boot=True
 pki_enable_proxy=False
 pki_proxy_http_port=80
 pki_proxy_https_port=443
-pki_security_manager=true
+pki_security_manager=false
```

Comment:
We will need to add an upgrade script to disable the security manager in existing instances (e.g. created in F42 then upgraded to F43). That can be added separately later.